spring-pulsar/src/test/java/org/springframework/pulsar/listener/ConcurrentPulsarMessageListenerContainerTests.java

```diff
@@ -41,28 +42,30 @@
 
 /**
  * @author Soby Chacko
+ * @author Alexander Preuß
  */
 public class ConcurrentPulsarMessageListenerContainerTests {
 
 	@Test
 	@SuppressWarnings("unchecked")
```

Comment:
**[NIT]** If we end up having the same suppression on all of the test methods, may just bring it up to the class level. Wdyt?

Reply:
I just saw that due to extracting the setup method, only `basicConcurrencyTesting` still requires suppression. So either we have it on both that and the `setupMockEnvironment` method or on the class level. I'm fine with both

Reply:
I prefer it fine-grain unless it is on ever single method. In this case I would prefer on those 2 places (setupMock + basicCon..). And, truly, a total NIT either way. 😄